src/scripts/main.js

```diff
@@ -1,3 +1,185 @@
+      const nameA = replace(a);
+      const nameB = replace(b);
+
+      if (!isNaN(nameA)) {
+        return order === 'asc' ? nameA - nameB : nameB - nameA;
+      } else {
+        return order === 'asc' ? nameA.localeCompare(nameB)
+          : nameB.localeCompare(nameA);
+      }
+    }).forEach(row => list.appendChild(row));
+  });
+});
+
+list.addEventListener('click', e => {
+  const current = e.target.closest('tr');
+
+  if (!current) {
+    return;
+  };
+
+  rows.forEach(row => {
+    if (row === current) {
+      row.classList.toggle('active');
+    } else {
+      row.classList.remove('active');
+    }
+  });
+});
+
+table.insertAdjacentHTML('afterend', `
+  <form class="new-employee-form">
+    <label>Name:
+      <input
+        name="name"
+        type="text"
+        data-qa="name"
+    ></label>
+    <label>Position:
+      <input
+        name="position"
+        type="text"
+        data-qa="position"
+    ></label>
+    <label>Office:
+      <select 
+        name="position"
+        data-qa="office"
+      >
+        <option selected disabled hidden></option>
+        <option>Tokyo</option>
+        <option>Singapore</option>
+        <option>London</option>
+        <option>New York</option>
+        <option>Edinburgh</option>
+        <option>San Francisco</option>
+      </select>
+    </label>
+    <label>Age:
+      <input
+        name="age"
+        type="number"
+        data-qa="age"
+    ></label>
+    <label>Salary:
+      <input
+        name="salary"
+        type="number"
+        data-qa="salary"
+    ></label>
+    <button type="submit">Save to table</button>
+  </form>
+`);
+
+const form = document.querySelector('.new-employee-form');
+const inputs = form.querySelectorAll('input[data-qa]');
+const select = form.querySelector('select[data-qa]');
+
+form.addEventListener('submit', e => {
+  e.preventDefault();
+
+  const values = {};
+
+  inputs.forEach(input => {
+    values[input.getAttribute('data-qa')] = input.value;
+  });
+  values[select.getAttribute('data-qa')] = select.value;
+
+  const nameEmployee = values['name'];
+  const positionEmployee = values['position'];
+  const officeEmployee = values['office'];
+  const ageEmployee = parseInt(values['age']);
+  const salaryEmployee = parseInt(values['salary']);
+
+  if (nameEmployee.length < 4) {
+    return showNotification('error',
+      'The name must contain at least 4 letters.');
+  }
+
+  if (positionEmployee.length === 0) {
+    return showNotification('error', 'The "Position" is empty.');
+  }
+
+  if (!officeEmployee) {
+    return showNotification('error', 'The Office field is not selected.');
+  }
+
+  if (isNaN(ageEmployee) || ageEmployee < 18 || ageEmployee > 90) {
+    return showNotification('error',
+      'The age must be a valid number between 18 and 90.');
+  }
+
+  if (isNaN(salaryEmployee) || salaryEmployee < 0) {
+    return showNotification('error', 'The "Salary" is empty.');
+  } else {
+    const newRow = list.insertRow();
```

Comment:
no needs to use `else` if you use early `return`
```suggestion
  if (isNaN(salaryEmployee) || salaryEmployee < 0) {
    return showNotification('error', 'The "Salary" is empty.');
  } 
  
  const newRow = list.insertRow();
```